examples/api/e2e/base.spec.ts

```diff
@@ -42,3 +42,9 @@ test("returns correct information about the request from a route handler", async
   const expectedURL = expect.stringMatching(/https?:\/\/localhost:(?!3000)\d+\/api\/request/);
   await expect(res.json()).resolves.toEqual({ nextUrl: expectedURL, url: expectedURL });
 });
+
+test("generates an og image successfully", async ({ page }) => {
+  const res = await page.request.get("/og");
+  expect(res.status()).toEqual(200);
+  expect(res.headers()["content-type"]).toEqual("image/png");
```

Comment:
aws uses [md5 checksum](https://github.com/opennextjs/opennextjs-aws/blob/a3337e04da43e3ea4c421471b318964f985ea729/packages/tests-e2e/tests/appRouter/og.test.ts#L5) in their tests, maybe something to add here